Castanet/Feed.php

```diff
@@ -273,6 +349,72 @@ protected function buildManagingEditor(DOMNode $parent)
 	}
 
 	// }}}
+	// {{{ protected function buildItunesOwner()
+
+	protected function buildItunesOwner(DOMNode $parent)
+	{
+		if ($this->itunes_email != '' || $this->itunes_owner != '') {
+			$document = $parent->ownerDocument;
+
+			$node = $document->createElementNS(
+				Castanet::ITUNES_NAMESPACE,
+				'owner'
+			);
+
+			if($this->itunes_email!= ''){
+				$text = $document->createTextNode($this->itunes_email);
+				$child_node = $document->createElementNS(
+					Castanet::ITUNES_NAMESPACE,
+					'email'
+				);
+				$child_node->appendChild($text);
+				$node->appendChild($child_node);
+			}
+
+			if($this->itunes_owner!= ''){
+				$text = $document->createTextNode($this->itunes_owner);
+				$child_node = $document->createElementNS(
+					Castanet::ITUNES_NAMESPACE,
+					'name'
+				);
+				$child_node->appendChild($text);
+				$node->appendChild($child_node);
+			}
+
+			$parent->appendChild($node);
+		}
+	}
+
+	// }}}
+	// {{{ protected function buildItunesCategory()
+
+	protected function buildItunesCategory(DOMNode $parent)
+	{
+		if ($this->itunes_category != '') {
+			$document = $parent->ownerDocument;
+
+			$node = $document->createElementNS(
+				Castanet::ITUNES_NAMESPACE,
+				'category'
+			);
+			$node->setAttribute('text', $this->itunes_category); 
+
+
+			if($this->itunes_subcategory!= ''){
```

Comment:
Multiple subcategories are valid according to the spec. Add support for an array:

```php
foreach ($this->itunes_subcategories as $subcategory) {
   // ...
}
```